packages/website/ts/components/documentation/docs_logo.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from '@0xproject/react-shared';
+import * as React from 'react';
+import { WebsitePaths } from 'ts/types';
+
+export interface DocsLogoProps {
+    height: number;
+    containerStyle?: React.CSSProperties;
+}
+
+export const DocsLogo: React.StatelessComponent<DocsLogoProps> = props => {
+    return (
+        <div style={props.containerStyle}>
```

Comment:
can we remove this wrapping div and just use `Container` when we want to alter the position of the `DocsLogo` which is what I assume is going on here?